buf.yaml

```diff
@@ -10,6 +10,8 @@ lint:
     - UNARY_RPC
     - COMMENT_FIELD
     - PACKAGE_DIRECTORY_MATCH
+    - ENUM_VALUE_PREFIX
+    - ENUM_ZERO_VALUE_SUFFIX
```

Comment:
What is the rationale for ignoring these?

Reply:
I would hold off on reviewing till tomorrow. This PR isn't ready for review.

Reply:
Sorry... I was requested for review...

Reply:
yes yes, sorry for that, it was done premature. Just meant to comment so you dont spend more time on it. 😄

Reply:
removed these, no need for them